src/end-chat/end-chat-styles.tsx

```diff
@@ -18,87 +18,77 @@ import * as FlexWebChat from '@twilio/flex-webchat-ui';
 
 const { styled } = FlexWebChat;
 
-export const EndChatWrapper = styled('div')`
-  margin: 2px auto;
-  width: 100%;
-  display: flex;
+export const EndChatWrapper = styled('span')`
```

Comment:
I see that there are a lot of changes to the styling. I think it will be helpful to account the spacing for other languages. For instance, using the ukr translation will break the spacing styling in this PR. Also, the previous requirement was to move the exit chat button down if the string is long. 
![Screenshot 2023-04-05 at 10 50 21 AM](https://user-images.githubusercontent.com/102122005/230118771-9817e86c-c8a7-4e45-8f1c-ceef7440275c.png)

Reply:
@mythilytm Great observation.